Dockerfile

```diff
@@ -1,45 +1,52 @@
-FROM perl:5.38
-
-RUN apt-get update && apt-get install -y \
-  bsd-mailx \
-  msmtp-mta \
-  pigz \
-  ruby-dev
-
-RUN cpanm -n  \
-  Data::Dumper \
-  Date::Manip \
-  DBD::MariaDB \
-  DB_File \
-  DBI \
-  Devel::Cover \
-  Devel::Cover::Report::Coveralls \
-  Exporter \
-  File::Slurp \
-  https://github.com/hathitrust/progress_tracker.git@v0.9.0 \
-  JSON::XS \
-  LWP::Simple \
-  Mail::Mailer \
-  MARC \
-  MARC::Batch \
-  MARC::File::XML \
-  MARC::Record \
-  MARC::Record::MiJ \
-  Sys::Hostname \
-  Test::More \
-  Test::Output \
-  URI::Escape \
-  XML::LibXML \
-  XML::LibXSLT \
-  YAML \
-  YAML::XS
+FROM debian:bookworm
 
 ENV ROOTDIR /usr/src/app
 ENV HOME /usr/src/app
 
 COPY . $ROOTDIR
 WORKDIR $ROOTDIR
 
+# Install Debian packages
+RUN apt-get update && apt-get install -y \
+    bsd-mailx \
+    build-essential \
+    cpanminus \
+    git \
+    libcrypt-ssleay-perl \
+    libdb-dev \
+    libmariadb-dev \
+    libnet-ssleay-perl \
+    libssl-dev \
+    msmtp-mta \
+    perl \
+    pigz \
+    ruby-dev
+
+# Install perl modules
+RUN cpanm --notest \
+    DBD::MariaDB \
```

Comment:
Some of these could be installed from Debian packages, but I'm not too fussed about trying to get them all that way right now.

Reply:
How would I go about figuring that out?

Reply:
https://packages.debian.org/bookworm/perl/ also seems viable.

Reply:
Honestly I'm not sure of the best way to get a comprehensive list. You could look at what we're installing from Debian vs. CPAN for babel here -- https://github.com/hathitrust/babel/blob/main/Dockerfile#L6-L59 -- that may give some pointers. The list in feed may also be useful to look at -- some are installed via apt and then anything that didn't get is installed by dependencies listed in Makefile.PL (see here https://github.com/hathitrust/feed/blob/main/Makefile.PL and https://github.com/hathitrust/feed/blob/main/Dockerfile#L23-L44).

Reply:
I've switched all perl modules that I could find from being installed with `cpanm` to being installed with `apt`. Removed a couple of modules that already come with perl (`Data::Dumper`, `Exporter`, `Sys::Hostname`).